.github/workflows/test.yml

```diff
@@ -0,0 +1,43 @@
+name: Test
+on:
+  push: { branches: [ "master" ] }
+  pull_request: { branches: [ "master" ] }
+
+concurrency:
+  group: test-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        python: ["3.9", "3.10"]
```

Comment:
In the setup configuration, you state that the minimum requirement is python 3.6. As far as I understand this part, the test will be run with python 3.9 and 3.10. So maybe you simply want to use 3.9 as a requirement in the setup.

Reply:
True.